src/tests/ftest/cart/tests_common.h

```diff
@@ -51,11 +51,12 @@ struct test_options {
 	int		num_attach_retries;
 	bool		is_server;
 	bool		assert_on_error;
+	int		shutdown;
+	int		delay_shutdown_sec;
 };
 
 static struct test_options opts = { .is_initialized = false };
```

Comment:
Do not declare variable in .h files.  If 2 different files in an application includes this .h file, then each file would have its own copy of this opts structure, which is probably not what you want.

Reply:
This is done on purpose for each test file to include this common test framework/file only once.

Reply:
But it would still be cleaner (in my opinion) to only have structure definitions, defines, etc in the .h file and then create a separate test_common.c with common code and globals, etc.

Reply:
There is plan to do more thorough refactoring of tests common framework, but this is a bigger task than what we want to do in this PR. evenetually yes, the cart tests common framework should consist of header for function/api definitions and all globals/vars will be in .c file linked to tests